### PR TITLE
x86: Remove deprecated TR_EnableNewX86PrefetchTLH

### PR DESCRIPTION
**Pull request by BradleyWood**
Issue: eclipse-openj9/openj9#9038

### CODE REVIEW

**Comment by BradleyWood** [comment]
jenkins build all

File: compiler/runtime/Helpers.inc [hunk 1]
```diff
@@ -165,10 +165,9 @@ SETVAL(TR_X86interpreterUnresolvedStaticFieldSetterGlue,TR_FSRH+29)
...
 SETVAL(TR_X86interpreterUnresolvedFieldSetterGlue,TR_FSRH+31)
 SETVAL(TR_X86prefetchTLH,TR_FSRH+32)
-SETVAL(TR_X86newPrefetchTLH,TR_FSRH+33)
-SETVAL(TR_X86CodeCachePrefetchHelper,TR_FSRH+34)
-SETVAL(TR_X86interpreterUnresolvedConstantDynamicGlue,TR_FSRH+35)
-SETVAL(TR_X86interpreterStaticAndSpecialGlue,TR_FSRH+36)
+SETVAL(TR_X86CodeCachePrefetchHelper,TR_FSRH+33)
+SETVAL(TR_X86interpreterUnresolvedConstantDynamicGlue,TR_FSRH+34)
+SETVAL(TR_X86interpreterStaticAndSpecialGlue,TR_FSRH+35)
 SETVAL(TR_LXRH,TR_FSRH+37)
```
Comment (0xdaryl) [inline compiler/runtime/Helpers.inc:171]: TR_FSRH+36

**Review by 0xdaryl — CHANGES_REQUESTED** [review]

**Comment by BradleyWood** [comment]
jenkins build all

@0xdaryl See [force-push](https://github.com/eclipse/omr/compare/9acbde2ab372b4ee5cc81b8ba774f97ff6788b26..e217fec6b35e4a9a4f81918f7a9e14a4673f7ea9)

**Comment by BradleyWood** [comment]
macOS failure is #7181

**Review by 0xdaryl — APPROVED** [review]

**Pull request merged**